.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Feature Request
+about: Ask for missing features or improvements
+
+---
+
+#### Description
+<!-- Here describe what is your need, you use cases and explain why this
```

Comment:
"Describe here […]", "your use cases"

Reply:
"Please describe here your use case, why you need this feature, and why this feature would be important and/or necessary for RIOT"